src/Cake.Common.Tests/Unit/Tools/WiX/HeatRunnerTests.cs

```diff
@@ -0,0 +1,674 @@
+using System;
```

Comment:
@RLittlesII can you add the new license file header? You will find an example on all the other files in the repo. This is a new requirement after joining the .Net Foundation.